src/einsteinpy/plotting/fractal.py

```diff
@@ -0,0 +1,127 @@
+import numpy as np
```

Comment:
Please mention everything on the top of this module about what it is. For future. Why we added it